src/services/settings.js

```diff
@@ -28,16 +28,33 @@ export const Defaults = {
  * @property {String}  globalFormat - The global format for new Dinero objects (see {@link module:Dinero~toFormat toFormat} for format).
  * @property {String}  globalRoundingMode - The global rounding mode for new Dinero objects (see {@link module:Dinero~multiply multiply} or {@link module:Dinero~divide divide} for format).
  * @property {String}  globalFormatRoundingMode - The global rounding mode to format new Dinero objects (see {@link module:Dinero~toFormat toFormat} or {@link module:Dinero~toRoundedUnit toRoundedUnit} for format).
+ * @property {String}  globalExchangeRatesApi.endpoint - The global exchange rates API endpoint for new Dinero objects (see {@link module:Dinero~convert convert} for format).
+ * @property {String}  globalExchangeRatesApi.JSONPath - The global exchange rates API JSON path for new Dinero objects (see {@link module:Dinero~convert convert} for format).
```

Comment:
Do you think it might be a bit misleading to call this `JSONPath` since it doesn't seem to actually be [JSONPath](http://goessner.net/articles/JsonPath/index.html#e2)?

[Lodash has a similar functionality](https://lodash.com/docs/4.17.10#property) which takes arrays into account which we might consider using here so that there is a well-understood/tested interface.

Reply:
I did some testing and you can access arrays by using the index `'b.c.1'`, so that works for me, but I'm still not sure we should call it JSONPath just because of the ambiguity.

Reply:
You're totally right about the name ambiguity. I'll go for `propertyPath`.